domains/dev-shireff.json

```
@@ -0,0 +1,8 @@
+{
+  "owner": {
+    "email": "shireffn369@gmail.com"
```

```suggestion
  "owner": {
  "username":"shireff",
    "email": "shireffn369@gmail.com"
```
Github username is required.